packages/subgraph/src/appRegistry.ts

```diff
@@ -0,0 +1,19 @@
+import { App as AppContract } from '../generated/AppRegistry/App';
+import { Transfer as TransferEvent } from '../generated/AppRegistry/AppRegistry';
+import { App } from '../generated/schema';
+import { checkAndCreateAccount, intToAddress } from './utils/utils';
+
+export function handleTransferApp(event: TransferEvent): void {
+  let contract = AppContract.bind(intToAddress(event.params.tokenId));
+  // Create and save the account entity
+  checkAndCreateAccount(contract.owner().toHex());
+
+  // Create and save the protectedData entity
+  let app = App.load(contract._address.toHex());
+  if (!app) {
+    app = new App(contract._address.toHex());
+    app.addOnlyAppWhitelists = new Array<string>();
+  }
+  app.owner = contract.owner().toHex();
+  app.save();
+}
```

Comment:
this will index all the apps on iExec and all the accounts owning an app, this is maybe not what we want.

Reply:
Commit: 0763f4087f1d2b5b13ed26862503e5aaebf0620c